### PR TITLE
Met 411 create statistics bolt

### PR DESCRIPTION
**Pull request by helinek**

### CODE REVIEW

File: service/dps/storm/topologies/validation/src/test/java/eu/europeana/cloud/service/dps/storm/topologies/validation/topology/ValidationTopologyTest.java [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import eu.europeana.cloud.service.dps.storm.NotificationTuple;
 import eu.europeana.cloud.service.dps.storm.ParseTaskBolt;
 import eu.europeana.cloud.service.dps.storm.io.*;
+import eu.europeana.cloud.service.dps.storm.topologies.validation.topology.bolts.StatisticsBolt;
 import eu.europeana.cloud.service.dps.storm.topologies.validation.topology.bolts.ValidationBolt;
```
Comment (tarekkh) [inline service/dps/storm/topologies/validation/src/test/java/eu/europeana/cloud/service/dps/storm/topologies/validation/topology/ValidationTopologyTest.java]: The ValidationExecutionService has been deployed to the artifactory, and this changes include removing the default -with no parameters- constructor. So for now If I want to clean install -as I did- , my build will fail. and I can't run ValidationTopologyTest tests.

File: service/dps/storm/topologies/validation/src/main/java/eu/europeana/cloud/service/dps/storm/topologies/validation/topology/bolts/StatisticsBolt.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import eu.europeana.cloud.service.dps.storm.utils.CassandraNodeStatisticsDAO;
+import eu.europeana.validation.model.ValidationResult;
+import eu.europeana.validation.service.ValidationExecutionService;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.util.List;
+
+public class StatisticsBolt extends AbstractDpsBolt {
```
Comment (tarekkh) [inline service/dps/storm/topologies/validation/src/main/java/eu/europeana/cloud/service/dps/storm/topologies/validation/topology/bolts/StatisticsBolt.java]: Missing unit tests

File: service/dps/storm/topologies/validation/src/test/java/eu/europeana/cloud/service/dps/storm/topologies/validation/topology/ValidationTopologyTest.java [hunk 3]
```diff
@@ -214,6 +215,7 @@ private static void buildTopology() {
         ReadDatasetBolt readDataSetBolt = new ReadDatasetBolt(MCS_URL);
         ReadRepresentationBolt readRepresentationBolt = new ReadRepresentationBolt(MCS_URL);
         NotificationBolt notificationBolt = new NotificationBolt("", 1, "", "", "");
+        StatisticsBolt statisticsBolt = new StatisticsBolt("", 1, "", "", "");
         TestInspectionBolt endTest = new TestInspectionBolt();
```
Comment (tarekkh) [inline service/dps/storm/topologies/validation/src/test/java/eu/europeana/cloud/service/dps/storm/topologies/validation/topology/ValidationTopologyTest.java]: This test doesn't really tests the statistics as part of the validation topology, as it only assert expected notification tuples, There is nothing about the data inserted into the statistics tables.

File: service/dps/storm/topologies/validation/src/main/java/eu/europeana/cloud/service/dps/storm/topologies/validation/topology/bolts/StatisticsBolt.java (hunk 2 above)
Comment (helinek) [reply service/dps/storm/topologies/validation/src/main/java/eu/europeana/cloud/service/dps/storm/topologies/validation/topology/bolts/StatisticsBolt.java]: Done

File: service/dps/storm/topologies/validation/src/test/java/eu/europeana/cloud/service/dps/storm/topologies/validation/topology/ValidationTopologyTest.java (hunk 3 above)
Comment (helinek) [reply service/dps/storm/topologies/validation/src/test/java/eu/europeana/cloud/service/dps/storm/topologies/validation/topology/ValidationTopologyTest.java]: Tests with data are in StatisticsBoltTest. It should be enough as we agreed.

**Pull request merged**